### PR TITLE
feat(scene): accessible scene

### PR DESCRIPTION
**Pull request by straker**
Accessible scene API following the examples from #115.

### CODE REVIEW

**Comment by straker** [comment]
@Rayne, @sdfereday, @BoldBigflank would this new [scene](https://github.com/straker/kontra/blob/scene/src/scene.js) work for your use cases alright?

File: kontra.js [hunk 1]
```diff
@@ -3357,6 +3381,192 @@ class Quadtree {
...
+    /**
+     * The scenes children objects.
+     * @memberof Scene
+     * @property {Object[]} children
+     */
+    this.children = [];
+
+    // create an accessible DOM node for screen readers
+    // dn = dom node
+    const section = this._dn = document.createElement('section');
```
Comment (Rayne) [inline kontra.js]: How does this node help users depending on screen readers without optimizing the whole application for screen readers?

**Comment by Rayne** [comment]
How does the scene workflow look like? Do we have to handle scenes ourselves, e.g.

1. Create scene/s

2. Update visible scenes in Kontra's update event

3. Render visible scenes like described before

?

**Comment by straker** [comment]
I have an example of a simple game with two scenes: https://github.com/straker/kontra/blob/v7/examples/scene. 

The basic workflow is you create a scene, add children to it, and then [update any active scenes in the main loop](https://github.com/straker/kontra/blob/v7/examples/scene/index.html). Hopefully this is similar to what you were doing with your scenes (minus the scene stack code). I was going to try to create a few more complex examples to see how I felt about it. But I think the basic structure is there.

File: kontra.js (hunk 1 above)
Comment (straker) [reply kontra.js]: The node is primarily for screen readers and keyboard users. I [wanted accessibility baked into the framework](https://github.com/straker/kontra/issues/124) if it started adding UI elements.

**Comment by Rayne** [comment]
I missed the examples. Sorry. :-)

I will have a look at it in the next days. Thanks.

**Comment by sdfereday** [comment]
Sorry @straker just seen this. Had a look through the examples and it all looks pretty darn good. It's a lot cleaner than what I ended up with so I'll switch over to it and see what happens from there.

**Pull request merged**